pyproject.toml

```
@@ -46,16 +48,16 @@ test = [
     "pytest-xdist",
     "filecheck",
 ]
-test-cu11 = [
-    "numba-cuda[cu11]",
-    "numba-cuda[test]",
-    "nvidia-curand-cu11",
-]
 test-cu12 = [
     "numba-cuda[cu12]",
     "numba-cuda[test]",
     "nvidia-curand-cu12",
 ]
+test-cu13 = [
+    "numba-cuda[cu13]",
+    "numba-cuda[test]",
+    "nvidia-curand==10.4.*",
```

This doesn't allow floating forward in minor versions for cuRAND. Should we make this `10.*` instead?

---

We could also use the [brand-new `cuda-toolkit` metapackage](https://pypi.org/project/cuda-toolkit/#description) to get a curand that ships with any CUDA 13 toolkit, like this:
```suggestion
    "cuda-toolkit[curand]==13.*",
```

---

I got the impression that 10.3 was for CUDA 12 and 10.4 for CUDA 13 so I wasn't sure about making it 10.*. Is that a misunderstanding?

Does `cuda-toolkit[curand]==13.*` install only curand?

---

Yes, see https://pypi.org/project/cuda-toolkit/ for the description. Various packages can be selected via extras, like `cuda-toolkit[curand,cublas,nvcc]`.

---

I'll add a TODO for this.